octokit-project/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
```

Comment:
[Severity: Low] The package declaration is present, but it is good practice to ensure that the package name follows the standard naming conventions, which typically use all lowercase letters.
Consider renaming the package to 'org.example' to maintain consistency with Java naming conventions.
Rationale: This helps in avoiding potential conflicts with other packages and improves readability.
Reference: Java Code Conventions recommend using lowercase for package names.